src/eventParsers/parseRdpcProgramUpdateEvent.ts

```diff
@@ -0,0 +1,32 @@
+const parseRdpcProgramUpdateEvent = (
+  message: string
+): RdpcProgramUpdateEvent => {
+  const obj = JSON.parse(message);
+  if (isProgramUpdateEvent(obj)) {
+    return obj;
+  } else {
+    throw new Error("Not a program update event");
+  }
+};
+
+type RdpcProgramUpdateEvent = {
+  studyId: string;
+  state: string;
```

Comment:
maybe make this an enum or union of string literal types? cause all the possible values are known eh